<launch>

	<!--open up the camera -->
	<node name="cv_camera_node" pkg="cv_camera" type="cv_camera_node" />
    
    <!-- setup the Arduino serial -->
    <node name="serial_node" pkg="rosserial_python" type="serial_node.py">
    	<param name="port" value="/dev/ttyACM0" />
    </node>

    <node name="steering_controller" type="steering_controller" pkg="steering_control">
    	<param name="/model_path" value="/home/neil/Workspace/self-driving-golf-cart/src/steering_control/scripts/weights/own/trained-cai-v6.h5" />
    	<param name="image_height" value="480" />
    	<param name="image_width" value="640" />
    	<param name="scale_factor" value='30' />
  	</node>

  	<node name="steering_visualization" type="visualization" pkg="steering_control" />
    
</launch>
